src/components/dashboard/orders/OrderContainer.js

```diff
@@ -0,0 +1,205 @@
+import React, {Component, PropTypes} from 'react';
+import {connect} from 'react-redux';
+
+import {getUserInfo} from '../../../actions/userActions';
+import {getOrders, getRoasterOrders} from '../../../actions/roasterActions';
+import {addOrder, uploadImage, updateOrder} from '../../../actions/warehouseActions';
+import ActionUtil from '../../../actions/actionUtil';
+import Order from './Order';
+import Loading from '../../Loading';
+
+class OrderContainer extends Component {
+    constructor(props) {
+        super(props);
+
+        this.handleRowClickBind = this.handleRowClick.bind(this);
+
+        this.state = {
+            tags: [],
+            image: '',
+            status: {},
+            selected: '',
+            selectedTags: [],
+            eimage: '',
+            estatus: {},
+            send: false,
+            ordersReceived: false
+        };
+
+        this.editHandlers = {
+            name: this._ref('ename'),
+            onAddType: this.getAddType('estatus'),
+            bags: this._ref('ebags'),
+            size: () => {},
+            price: () => {},
+            isDecaf: this._ref('eisDecaf'),
+            isActive: this._ref('eisActive'),
+            description: this._ref('edescription'),
+            photo: this.handleEditPhoto.bind(this),
+            onAddBeans: this.handleEditBeans.bind(this)
+        };
+    }
+
+    componentDidMount() {
+        console.log(this.props);
+        this.props.dispatch(getOrders(this.props.roaster.id, 0, 100)).then(() => {
+            this.setState({showItems: true});
+        });
+    }
+
+    componentWillReceiveProps(next) {
+        if (next.modify.success && (this.state.send || this.state.esend)) {
+            this.handleSuccess();
+            return;
+        }
+
+        if (!this.props.roaster.id || !this.props.next) {
+            return;
+        }
+
+        this.props.dispatch(getRoasterOrders(this.props.roaster.id, 0, 100)).then(() => {
+            this.setState({ordersReceived: true});
+        });
+        // this.props.dispatch(getOrders(this.props.roaster.id, 0, 100)).then(() => {
+        //     this.setState({ordersReceived: true});
+        // });
+    }
+
+    handleSuccess() {
+        if (this.state.send) {
```

Comment:
we don't need all of these. orders aren't created by the roaster, and the only thing they should update id getting the tracking url.